cra-rxjs-styled-components/src/hooks/formatDistance/use-format-distance.ts

```diff
@@ -0,0 +1,29 @@
+import { formatDistance, Locale } from 'date-fns';
+
+const isValidDate = (date: Date | number): boolean =>
+  !(date === undefined || date === null);
+
+export type Props = {
+  date: Date | number | string;
+  dateToCompare: Date | number;
+  options?: {
+    includeSeconds?: boolean;
+    addSuffix?: boolean;
+    locale?: Locale;
+  };
+};
+
+export function useFormatDistance({
```

Comment:
Rename this file and function to something else. Using `use` in front of the name suggests that it is a custom hook. but it's not. Also, rename the file and move it out into the `helpers` folder. This is just a normal helper function to format dates.